canonicalwebteam/flask_base/app.py

```diff
@@ -103,9 +103,20 @@ def internal_error(error):
         def status_check():
             return os.getenv("TALISKER_REVISION_ID", "OK")
 
-        if favicon_url:
+        favicon_path = os.path.join(self.root_path, "../static", "favicon.ico")
+        if os.path.isfile(favicon_path):
 
             @self.route("/favicon.ico")
+            @self.route("/favicon.png")
```

Comment:
so here it can't be a png since you are only testing for the `.ico` file.

Reply:
there is a test for .png too

Reply:
but it exists because you give it a ico file and a png file (to the test server.) If you remove the ico file from your test and run the png should fail, because the if  (just above) statements is only checking for a ico file.

Reply:
the idea was to serve the same file for both routes, cause I misread the redirects and tought favicon.ico and favicon.png were redirecting to the same place, I'll add another route